python-sdk-sample/tests/unit/test_sample_sdk.py

```diff
@@ -0,0 +1,56 @@
+from unittest import TestCase, mock
+import unittest
+from zowe.core_for_zowe_sdk import ProfileManager # type: ignore
+
+import sys
+import os
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(current_dir, "../../src"))
+from main import SampleSdk
+
+class TestSampleSDK(TestCase):
+
+    @mock.patch("requests.Session.send")
+    def test_create_post(self, mock_send_request):
+        mock_send_request.return_value = mock.Mock(headers={"Content-Type": "application/json"}, status_code=201)
+        profile = ProfileManager().load(profile_type="zosmf", validate_schema=False)
```

Comment:
If the above comments are addressed, we shouldn't need to load z/OSMF profiles in unit tests anymore.